<?xml version="1.0"?>
<launch>

  <arg name="type_info" default="pose"/>  
  <arg name="pose_topic" default="/master_phantom/pose"/>
  <arg name="vel_topic" default="/master_phantom/velocity"/>
  <arg name="read_ip" default="192.168.0.108"/>
  <arg name="read_port" default="34900"/>
  <arg name="publish_rate" default="1000.0"/>	
  <arg name="ref_frame" default="world"/>
   
  <!-- Receive UDP Node -->
  <node name="receive_udp_pose" pkg="teleop_strategies_controller" type="receive_udp_pose.py" output="screen"> 
	<param name="pose_topic" type="string" value="$(arg pose_topic)"/>
	<param name="vel_topic" type="string" value="$(arg vel_topic)"/>
    <param name="read_ip" type="string" value="$(arg read_ip)"/>
    <param name="read_port" type="int" value="$(arg read_port)"/>
    <param name="publish_rate" type="double" value="$(arg publish_rate)" />
    <param name="ref_frame" type="string" value="$(arg ref_frame)"/>
  </node>
  
</launch>
